test/Stove.Migrator.Tests.Domain/Migrations/Migrations/InitialTestStoveMigration.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Stove.Migrator.Tests.Domain.Migrations.Migrations
 {
-    [StoveVersionInfo(author: "Ayberk CANTÜRK", description: "Initial Test Stove Migration", version: "201705071543")]
+    [StoveVersionInfo("Ayberk CANTÜRK", "Initial Test Stove Migration", "201705071543", Enviroments.Dev, Enviroments.Stage, Enviroments.Production)]
```

Comment:
Ok I am going to implement it as a new attribute type and allow user to pass multiple tags.